Settings (#42)
ModellingWebLab/deployment: README.md

cd deployment
git submodule update --init
```

> ### Running in a folder
> If you're running weblab in a folder e.g. http://myhost.com/weblab/ then edit `roles/django/templates/deployed.j2` and add the following lines with the correct folder name e.g.:

Reviewer: ```suggestion
### Running at a sub-URL

If you're hosting WebLab at a sub-URL on a domain (e.g. http://myhost.com/weblab/ rather than just http://myhost.com/) then edit `roles/django/templates/deployed.j2` and add the following lines with the correct URL prefix name, e.g.:
```

```
FORCE_SCRIPT_NAME = '/weblab/'

> STATIC_URL =+ '/static/weblab/'

Reviewer: Shouldn't this be `/weblab/static/`?

Author: you're quite right of ocurse

LOGIN_REDIRECT_URL = '/weblab/'
LOGOUT_REDIRECT_URL = '/weblab/'
```

> 

Reviewer: ```suggestion

Note that you will need to set up the appropriate proxying *outside* the Web Lab deployment so that requests to your sub-URL come through to the nginx server used by Web Lab. Such configuration is outside the scope of this guide.

```

```shell
vagrant up --provision
```

> #### Add initial admin/superuser

Reviewer: ```suggestion

#### Add initial admin/superuser
```
